test.cpp

```diff
@@ -0,0 +1,19 @@
+#include "Pusher.h"
+
+int main()
+{
+    Pusher pusher("ck.pem", "123456");

```

Comment:
This is my fault, I did twice modifications, one is that add password for pem, another is that I add a sample for myself. After my first modification, I committed to my fork and requested a pull to your repository. So I thought the rest of modifications will keep in my fork if didn't request a pull to your git respository. 
Actually, when I saw this page that two changes appeared, I feel so unprofessional. Anyway, thank you for merging my requestion and the correction of naming.